ci/transformers_integration_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
another thing I just realized was this script was not added to our CI on github actions, you can add a new task with this script to this config file: https://github.com/bentoml/BentoML/blob/master/.github/workflows/ci.yml

Reply:
Sorry I guessed I missed that. I'll add it up in another PR